src/components/profile/ProfileInfo.jsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import { useFetchUserQuery } from '../../redux/slices/userSliceApi';
+import { axiosInstance } from '../../axios';
+import { toast } from 'react-toastify';
+
+import Chesterna from '../icons/Chesterna';
+import ProfilePhoto from './ProfilePhoto';
+import Modal from '../shared/Modal';
+import Button from '../shared/Button';
+
+const ProfileInfo = ({ completed_tasks, created_tasks }) => {
+  const { data, isLoading } = useFetchUserQuery(localStorage.getItem('user_id'));
+
+  const [avatarFormOpen, setAvatarFormOpen] = useState(false);
+  const [selectedFile, setSelectedFile] = useState(null);
+
+  const handleUploadUserAvatar = async () => {
+    if (!selectedFile) {
+      alert('please select a file');
+      return;
+    }
+
+    let formData = new FormData();
+    formData.append('file', selectedFile);
+    formData.append('user_id', localStorage.getItem('user_id'));
```

Comment:
** Suggestion ** 
same here about user_id, get it once when you create a component and get where you need it please